services.py

```diff
@@ -1,44 +1,32 @@
-import os
+import json
 import requests
 import subprocess
 import configparser
 import geocoder
 import ast
 import logging
-from geopy.geocoders import Nominatim
+from geopy.geocoders import Photon
+import serial
+import darkdetect
+import screen_brightness_control as sbc
+
+
+SERIAL_PORT = "/dev/ttyUSB0"
+
+
 
 class Services:
     def __init__(self):
-        self.logger = self.get_logger('services')
+        self.logger = logging.getLogger('services')
```

Comment:
you should still initialize the loggeer in the global scope, so it is 1 logger per file, not 1 logger per class instance. Even though you shouldn't have more than 1 instance already, this is how loggers are expected to look like in Python.